app/src/main/java/com/keylesspalace/tusky/BaseActivity.java

```diff
@@ -212,15 +212,8 @@ public void showAccountChooserDialog(CharSequence dialogTitle, boolean showActiv
         }
     }
 
-    // TODO: This changes the accountManager's activeAccount property, but does not do any
-    // of the work that AccountManager.setActiveAccount() does. In particular:
-    //
-    // - The current active account is not saved
-    // - The account passed as parameter here goes not have its `isActive` property set
-    //
-    // Is that deliberate? Or is this a bug?
```

Comment:
It is definitely a bug, the setter should have never been public, and was probably only called because it looks the same as the method from Java. I tested it, works fine with this change.